### ISSUE TITLE
fake initials on t7.1 generates future deprecation errors

### ISSUE THREAD

**Issue by eschipul** [tendenci/tendenci#439]
When running fake initials on an upgrade of 8 it generates numerous future deprecation errors. Adding this issue to track those for milestone tendenci 8 which will be on Django 1.9 (or possibly still 1.8 if 1.9 isn't LTS.) Just capturing the data here.

```
root@tendenci:/var/www/tendenci# python manage.py fake_initials
/usr/local/lib/python2.7/dist-packages/captcha/models.py:36: RemovedInDjango19Warning: Model class captcha.models.CaptchaStore doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class CaptchaStore(models.Model):

/usr/local/lib/python2.7/dist-packages/django/contrib/admin/util.py:7: RemovedInDjango19Warning: The django.contrib.admin.util module has been renamed. Use django.contrib.admin.utils instead.
  "Use django.contrib.admin.utils instead.", RemovedInDjango19Warning)

/usr/local/lib/python2.7/dist-packages/tendenci/apps/events/forms.py:16: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils.importlib import import_module

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 351, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 325, in execute
    django.setup()
  File "/usr/local/lib/python2.7/dist-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/dist-packages/django/apps/registry.py", line 89, in populate
    "duplicates: %s" % app_config.label)
django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: testimonials
```

Ran into this error when running the commands here:
https://tendenci.readthedocs.org/en/latest/upgrade/upgrade-to-tendenci7.html#step-5-fake-initials

**Comment by PaulSD** [comment]
Fixed in #659

**Comment by eschipul** [comment]
NA
